README.md

```diff
@@ -11,66 +11,122 @@ pages with [mkdocs-material](https://squidfunk.github.io/mkdocs-material/).
 
 ## How to do a Quick Edit on github
 
-To do a quick edit, use this is a 3 step process:
+To do a quick edit, use this is a 4 step process:
 
```

Comment:
This should be a real bulleted list. Easier to read than a bunch of one-sentence lines.